deployments/qat_plugin/overlays/debug/add-args.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: intel-qat-plugin
+spec:
+  template:
+    spec:
+      containers:
+      - name: intel-qat-plugin
+        args:
+        - "-v=4"
```

Comment:
```-v = 4``` is set by default in https://github.com/intel/intel-device-plugins-for-kubernetes/blob/main/pkg/controllers/qat/controller.go#L226
We should either make it default in both deployment scenarios or make it optional in controller.go I guess.

Reply:
there are a lot more flags handled by controller.go. Should those be somehow reflected in the kustomize yamls?

Reply:
> Should those be somehow reflected in the kustomize yamls?

IIRC we just provided the `debug` overlay as an example/helper overlay. We don't need more than that.